cpp/python/src/notifier.cpp

```diff
@@ -23,16 +23,17 @@ Notifier::~Notifier() {}
 
 void Notifier::scheduleFutureNotify(std::shared_ptr<::ucxx::Future> future, ucs_status_t status)
 {
-  ucxx_trace_req(
-    "Notifier::scheduleFutureNotify(): future: %p, handle: %p", future.get(), future->getHandle());
+  ucxx_trace_req("ucxx::python::Notifier::scheduleFutureNotify, future: %p, handle: %p",
```

Comment:
Note to switch to `std::source_location::current().function_name()` once using C++20. Although that does seem to include the signature, which might not be ideal :(.

Reply:
Yeah, exactly, plus we're still on C++17 for now at least. 🙁 

Not the perfect solution but we can use `__func__` for at least part of this, done in https://github.com/rapidsai/ucxx/pull/156/commits/124efaedeee0b1e1b0c32d7424c7272a06423b92 .